test/common/config/utility_test.cc

```diff
@@ -295,6 +296,65 @@ TEST(UtilityTest, PrepareDnsRefreshStrategy) {
   }
 }
 
+void packTypedStructIntoAny(ProtobufWkt::Any& typed_config, const ProtobufWkt::Message& inner) {
+  udpa::type::v1::TypedStruct typed_struct;
+  (*typed_struct.mutable_type_url()) =
+      std::string("type.googleapis.com/") + inner.GetDescriptor()->full_name();
+  MessageUtil::jsonConvert(inner, *typed_struct.mutable_value());
+  typed_config.PackFrom(typed_struct);
+}
+
+TEST(UtilityTest, TypedStructToStruct) {
```

Comment:
Can you add one liner `//` above these tests explaining what htey do?

Reply:
done